src/main/java/org/mifos/processor/bulk/zeebe/worker/AccountLookupWorker.java

```diff
@@ -68,13 +75,27 @@ public void setup() {
             exchange.setProperty(HOST, identityMapperURL);
             exchange.setProperty(PAYEE_IDENTITY, payeeIdentity);
             exchange.setProperty(PAYMENT_MODALITY, paymentModality);
-            exchange.setProperty(CALLBACK, bulkURL + "/accountLookup/Callback");
+            exchange.setProperty(CALLBACK, identityMapperURL + "/accountLookup/callback");
             exchange.setProperty(TRANSACTION_ID, existingVariables.get(TRANSACTION_ID));
             exchange.setProperty("requestId", job.getKey());
             exchange.setProperty(CHANNEL_REQUEST, channelRequest);
             exchange.setProperty(ORIGIN_DATE, existingVariables.get(ORIGIN_DATE));
             exchange.setProperty(TENANT_ID, tenantId);
-            producerTemplate.send("direct:send-account-lookup", exchange);
+            exchange.setProperty(HEADER_REGISTERING_INSTITUTE_ID, existingVariables.get(HEADER_REGISTERING_INSTITUTE_ID));
+
+            RequestSpecification requestSpec = new RequestSpecBuilder().build();
+            requestSpec.relaxedHTTPSValidation();
+
+            Response response = RestAssured.given(requestSpec)
```

Comment:
can you move all external calls to one service and call that from here?
It will allow us to handle certificates/ logs/ authentication etc etc

Reply:
Also I prefer not to bring multiple libraries for calling rest API, we might already be using one pattern ( Webclient / rest template ) in the service

Reply:
I am now using camel to camel to call external API this is already being used to call channel APIs